CMakeLists.txt

```diff
@@ -435,114 +480,84 @@ function(is_sdk_requested name result_var_name)
   endif()
 endfunction()
 
-# FIXME: separate the notions of SDKs used for compiler tools and target
-# binaries.
-if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
-  set(CMAKE_EXECUTABLE_FORMAT "ELF")
+# Required from this point:
+# - SWIFT_HOST_VARIANT_SDK  - OS of system which will host the compiler
+# - SWIFT_HOST_VARIANT_ARCH - arch of system which will host the compiler
+# 
+# FIXME: cross-compile standard libarary targets are specified by "SDK",
+# which locks each one to the hardcoded architectures defined here (e.g. armv7 for ANDROID)
```

Comment:
I see.  This is something that we actually want even for iOS, to be able to enable/disable either SDK as a whole, or just particular slices.  Could you paste your comment into the patch as the FIXME text?